test/integration/features/experimental-feature.feature

```diff
@@ -0,0 +1,47 @@
+@experimental-feature
+Feature: Experimental Feature
```

Comment:
This should be in plural or specify that the feature tests service-catalog: `service-catalog.feature`.

Reply:
@agajdosi agree `s/Feature/Features/`